I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scenes_script/scene_3.rpy", line 58, in script
    show doctor_icon centered
Exception: Image 'doctor_icon' does not accept attributes 'centered'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scenes_script/scene_3.rpy", line 58, in script
    show doctor_icon centered
  File "/Volumes/renpy-7.5.3-sdk/renpy-7.5.3-sdk/renpy/ast.py", line 1408, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "/Volumes/renpy-7.5.3-sdk/renpy-7.5.3-sdk/renpy/ast.py", line 1378, in show_imspec
    atl=atl)
  File "/Volumes/renpy-7.5.3-sdk/renpy-7.5.3-sdk/renpy/exports.py", line 686, in show
    if not base.find_target() and renpy.config.missing_show:
  File "/Volumes/renpy-7.5.3-sdk/renpy-7.5.3-sdk/renpy/display/image.py", line 414, in find_target
    self.target = target._duplicate(a)
  File "/Volumes/renpy-7.5.3-sdk/renpy-7.5.3-sdk/renpy/display/core.py", line 397, in _duplicate
    args.extraneous()
  File "/Volumes/renpy-7.5.3-sdk/renpy-7.5.3-sdk/renpy/display/core.py", line 267, in extraneous
    " ".join(self.args),
Exception: Image 'doctor_icon' does not accept attributes 'centered'.

Darwin-21.6.0-x86_64-i386-64bit x86_64
Ren'Py 7.5.3.22090809
BetaGame 1.0
Wed Oct 19 10:35:53 2022
